src/main/java/org/jenkinsci/plugins/docker/commons/credentials/DockerServerEndpoint.java

```diff
@@ -78,12 +82,21 @@ public DockerServerEndpoint(String uri, String credentialsId) {
      *
      * <p>
      * Null to indicate whatever Docker picks by default.
+     * @deprecated use #getDockerHost
      */
     public @Nullable String getUri() {
         return uri;
     }
 
     /**
+     * Gets the endpoint in URI, such as "unix:///var/run/docker.sock".
+     */
+    public @Nonnull String getDockerHost() {
```

Comment:
I used "dockerhost" to map the DOCKER_HOST environment variable used for CLI usage